### PR TITLE
issue146

### LINKED ISSUE #146: Cyclic dependency between PharoLauncher-Core and PharoLauncher-Download
There is a cyclic dependency between PharoLauncher-Core and PharoLauncher-Download because of PhLError and PhLTObject

### PR DESCRIPTION
**Pull request by pavel-krivanek**
Fix #146: move PhLError and PhLTObject to PharoLauncher-Core

### CODE REVIEW

**Comment by demarey** [comment]
@pavel-krivanek take care: I think @VincentBlondeau wants PharoLauncher-Download to be independent. He uses it in Cruiser.

**Comment by tesonep** [comment]
Fix #146

**Comment by guillep** [comment]
@demarey let's make it work as a project first. People should not download independent packages like that or they may break everything. I'd say: If you want `PharoLauncher-Download` you'll get `PharoLauncher-Core` too. That's a pretty fair trade off...

File: src/PharoLauncher-Core.package/monticello.meta/categories.st [hunk 1]
```diff
@@ -2,3 +2,5 @@ SystemOrganization addCategory: #'PharoLauncher-Core'!
 SystemOrganization addCategory: 'PharoLauncher-Core-Commands'!
 SystemOrganization addCategory: 'PharoLauncher-Core-Model'!
 SystemOrganization addCategory: 'PharoLauncher-Core-Settings'!
+el'!
+SystemOrganization addCategory: #'PharoLauncher-Core-Settings'!
```
Comment (guillep) [inline src/PharoLauncher-Core.package/monticello.meta/categories.st]: This looks broken :/

**Comment by pavel-krivanek** [comment]
It was already this case because PharoLauncher-Download depends on the Core. Maybe we can separate it to a standalone project but later

**Comment by pavel-krivanek** [comment]
I fixed it manually, it looks like an issue with the new streams - it rewrote the file without touching o the original content. I have seen similar issue during some file-outs.

**Comment by guillep** [comment]
I'll integrate it, otherwise I have problems with circular dependencies while creating a baseline

**Pull request merged**